Add a guide on how to use apps like functions (#1799)
gradio-app/gradio: guides/using_apps_like_functions.md

<script type="module" src="https://gradio.s3-us-west-2.amazonaws.com/3.0.24/gradio.js"></script>

> # Using Gradio Apps Like Functions

Reviewer: How about:
```suggestion
# Using Gradio Blocks Like Functions
```

Docs: Blocks

**Prerequisite**: This Guide builds on the Blocks Introduction. Make sure to [read that guide first](/introduction_to_blocks).

## Introduction

> Did you know that apart from being a full-stack machine learning demo, a Gradio app is also a regular-old python function!?

Reviewer: ```suggestion
Did you know that apart from being a full-stack machine learning demo, a Gradio Blocks app is also a regular-old python function!?
```

> This means that if you have a gradio app called `demo`, you can use `demo` like you would any python function.

Reviewer: ```suggestion
This means that if you have a gradio Blocks (or Interface) called `demo`, you can use `demo()` like you would any python function.
```
